### PR TITLE
feat: Automatic release process

### PR DESCRIPTION
**Pull request by yashk2000**
Changes: Initial attempt for the auto release process.

### CODE REVIEW

**Comment by yashk2000** [comment]
@iamareebjamal @abishekvashok @sauravvishal8797 please review and tell what changes have to be made.

**Comment by iamareebjamal** [comment]
Not complete yet. Will make some changes.

**Comment by yashk2000** [comment]
@iamareebjamal please tell me what changes to make.

**Comment by iamareebjamal** [comment]
Add prep-key.sh and this in the travis build. I'll review and take it from there

**Comment by yashk2000** [comment]
@iamareebjamal done.

**Comment by iamareebjamal** [comment]
Last command (fastlane) is wrong

**Comment by yashk2000** [comment]
@iamareebjamal is it correct now?

**Comment by yashk2000** [comment]
@iamareebjamal is it correct now?

File: scripts/update-apk.sh [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	done
+fi
+
+# Signing Apps
+
+if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
+    echo "Push to master branch detected, signing the app..."
+    cp app-release-unsigned.apk app-release-unaligned.apk
+    jarsigner -verbose -tsa http://timestamp.comodoca.com/rfc3161 -sigalg SHA1withRSA -digestalg SHA1 -keystore ../scripts/key.jks -storepass $STORE_PASS -keypass $KEY_PASS app-release-unaligned.apk $ALIAS
+    ${ANDROID_HOME}/build-tools/26.0.3/zipalign -v -p 4 app-release-unaligned.apk app-release.apk
```
Comment (iamareebjamal) [inline scripts/update-apk.sh]: Wrong command

File: scripts/update-apk.sh [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+fi
```
Comment (iamareebjamal) [inline scripts/update-apk.sh]: You have to move this block before than mv commands

File: scripts/update-apk.sh (hunk 1 above)
Comment (yashk2000) [reply scripts/update-apk.sh]: @iamareebjamal could you please tell me what's wrong with the command here?

File: scripts/update-apk.sh (hunk 2 above)
Comment (yashk2000) [reply scripts/update-apk.sh]: Ok.

**Pull request merged**